pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -659,6 +659,40 @@ class Build {
         }
     }
 
+    def waitForANodeToBecomeActive(def label) {
+        def NodeHelper = context.library(identifier: 'openjdk-jenkins-helper@master').NodeHelper
+
+        if (NodeHelper.nodeIsOnline(label)) {
+            return
+        }
+
+        context.println("No active node matches this label: " + label)
+
+        int activeNodeTimeout = 0
+        if (${buildConfig.ACTIVE_NODE_TIMEOUT}.isInteger()) {
+            activeNodeTimeout = ${buildConfig.ACTIVE_NODE_TIMEOUT} as Integer
+        }
+
+
+        if (activeNodeTimeout > 0) {
+        context.println("Will check again periodically until either one comes online, or " + ${buildConfig.ACTIVE_NODE_TIMEOUT} + " minutes (ACTIVE_NODE_TIMEOUT) has passed.")
```

Comment:
Inline if statement

Reply:
I've always been taught that inline IF statements are bad practice, though I don't recall why. Thoughts?

Reply:
Inline as in the line underneath the if statement is not indented so that it's clearly inside the if statement

Reply:
Actually it is. Github diff display bug, I think.

See that same line here: https://github.com/adamfarley/openjdk-build/blob/timeout_when_no_nodes_found/pipelines/build/common/openjdk_build_pipeline.groovy

Reply:
Ah, k. Will fix.